hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: otbn
+  testpoints: [
+    {
+      name: chip_sw_otbn_isatest
+      descr: '''Run every OTBN instruction and check final state.
+
+                Run the existing OTBN smoke test which uses every instruction
+                (hw/ip/otbn/dv/smoke/smoke_test.s). Check the final register state matches the
+                expected values (in hw/ip/otbn/dv/smoke/smoke_expected.txt). Check OTBN completed
+                successfully without errors.
+
+                Following the smoke test run a new program just to dump out the register state to
+                dmem, check the values do not match the previous values from the smoke test run to
+                ensure the internal state secure wipe has worked.
+             '''
+      stage: V2
+      si_stage: SV1
+      tests: [""]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA",
+        "OTBN.SECUREWIPE
+      ]
+    }
+    {
+      name: chip_sw_otbn_op
+      desc: '''Verify an OTBN operation.
+
+            - SW test directs the OTBN engine to perform an ECDSA operation.
+            - SW validates the reception of the otbn done interrupt once the operation is complete.
+            - SW verifies the correctness of the result with the expected value which is
+              pre-computed using a reference model.
+            '''
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_sw_otbn_ecdsa_op_irq",
+              "chip_sw_otbn_ecdsa_op_irq_jitter_en"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA", "PROD", "PROD_END"]
+      features: [
+        "OTBN.ISA"
+      ]
+    }
+    {
+      name: chip_sw_otbn_rnd_entropy
+      desc: '''Verify OTBN can fetch RND numbers from the entropy src.
+
+            - SW initializes the entropy subsystem to generate randomness.
+            - SW loads an OTBN app that executes instructions to read the RND bits.
+            - The OTBN app ensures that the values when read consequtively do not match, and its not
```

Comment:
I don't think you want "its" here. Maybe "and that no value is all 0s or all 1s"?

Also, spelling of "consecutively"

Reply:
In my defence I didn't write this bit, came from the old testplan!  I'll fix it